assigner.php

```diff
@@ -0,0 +1,174 @@
+<?php
+require_once __DIR__ . "/include/header.php";
+require_once __DIR__ . "/include/tsutils.php";
+require_once __DIR__ . "/include/cacheutils.class.php";
+require_once __DIR__ . "/config/assignerconfig.php";
+
+$clientgroups = [];
+
+try {
+    $tsAdmin = getTeamspeakConnection("#no_query_clients");
+
+    $userip = $_SERVER["HTTP_CF_CONNECTING_IP"]; // $_SERVER["REMOTE_ADDR"];
+
+    echo "<!-- IP: {$_SERVER["REMOTE_ADDR"]}, CFIP: {$_SERVER["HTTP_CF_CONNECTING_IP"]} -->";
```

Comment:
The switch for real IP / proxy IP needs to not be hardcoded, rather added to the assigner config